cclib/method/ddec.py

```diff
@@ -544,20 +598,20 @@ def condition_densities(self):
                         lowerphi = self._candidates_phi[atomi][lower_ind]
                         upperphi = self._candidates_phi[atomi][upper_ind]
 
-                        if abs(lowerphi) < 1e-10:
+                        if abs(lowerphi) < self.convergence_level:
                             bigphiAI[atomi] = self._candidates_bigPhi[atomi][lower_ind]
-                            fitphi = False
-                        elif abs(upperphi) < 1e-10:
+                            phiconverged = True
+                        elif abs(upperphi) < self.convergence_level:
                             bigphiAI[atomi] = self._candidates_bigPhi[atomi][upper_ind]
-                            fitphi = False
+                            phiconverged = True
                         else:
                             # Fitting needs to continue in this case.
                             lowerbigPhi = self._candidates_bigPhi[atomi][lower_ind]
                             lowerphi = self._candidates_phi[atomi][lower_ind]
                             upperbigPhi = self._candidates_bigPhi[atomi][upper_ind]
                             upperphi = self._candidates_phi[atomi][upper_ind]
 
-            assert not fitphi, "Iterative conditioning failed to converge."
+            assert phiconverged, "Iterative conditioning failed to converge."
```

Comment:
This will raise `AssertionError`. Perhaps on `not phiconverged` raise a more appropriate error?

Reply:
I've added `ConvergenceError` here. Thanks.